### PR TITLE
#88 Create Login Page using Login component

### PR DESCRIPTION
**Pull request by laila1222**
- Created component of login form 
- Created login page - @rakhshanda-rashid

### CODE REVIEW

**Review by bytewiz — CHANGES_REQUESTED** [review]
There is a lot of outdated changes here and files that should not be changed e.g. all files with `git` should not be here please remove them and we don't use enzyme for testing we use `react-testing-library` and the formatting is also a but off and please don't change the jest configurations...

**Comment by bytewiz** [comment]
Please rename `FP-annotatetheweb-class10.git ` to `.git`

**Review by laila1222 — COMMENTED** [review]
I removed the files that shouldn't have been changed from this branch.

File: src/client/components/SignInImage/SignInImage.component.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const Image = ({classImage, loginImage}) => {
+  return (
+   <div>
+      <img className={classImage} src={loginImage}/>   
+  </div>
+  )
+};
+
+export default Image;
```
Comment (mo3ni) [inline src/client/components/SignInImage/SignInImage.component.jsx:14]: do we need to create separate component for just an image or just define it inside sign in page!

**Review by mo3ni — APPROVED** [review]

File: src/client/components/Header/Header.css [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
 
 .header {
-  position: absolute;
-  width: 1767px;
-  height: 105px;
-  left: 77px;
-  top: 50px;
+  /* weight: 13rem;
+  left: -10px;
+  top: 15px; */
```
Comment (mo3ni) [inline src/client/components/Header/Header.css]: remove comments

File: src/client/components/Header/Header.css [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
-  position: absolute;
-  width: 440px;
-  height: 105px;
-  left: 77px;
-  top: 50px;
-
+  width: 14rem;
+  /* height: px;
+  left: -2500px;
+  top: -35px; */
```
Comment (mo3ni) [inline src/client/components/Header/Header.css]: remove comments

File: src/client/components/Header/Header.css [hunk 4]
```diff
@@ -1,27 +1,30 @@
...
+  font-size: 20px;
+  /* line-height: 105px; */
+  padding-left: 2rem;
+}
 
+.logo {
+  font-family: Pacifico;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5rem;
```
Comment (mo3ni) [inline src/client/components/Header/Header.css:20]: 2 logo css property with different values

File: src/client/components/SignInImage/SignInImage.component.jsx (hunk 1 above)
Comment (laila1222) [reply src/client/components/SignInImage/SignInImage.component.jsx:14]: Done!

**Review by bytewiz — APPROVED** [review]
Think that should it 👍

**Pull request merged**